CONTRIBUTING.rst

```diff
@@ -111,6 +111,26 @@ Please note that there is a minimum review period of 24 hours for any patch. The
 
 All subsequent changes made to patches must be force-pushed to the PR branch before merging it into the main branch.
 
+Code conventions
+----------------
+
+It is important to have consistency across the codebase. This won't necessarily make your code work better, but it might help make it more understandable, time-consuming and less irritating to go through when doing a code review of your changes.
```

Comment:
`but it might help make it more understandable, time-consuming and less irritating to go through when doing a code review of your changes.`

I think we should reword this:

`but it might help make he codebase more understandable, easier to work with and more pleasant to go through when doing a code review.`

This should address:
- removes the implication that following code convention makes working with the codebase more time consuming
- removes the implication that code review is irritating ;-)

Reply:
Fixed.